PICMI_Python/setup.py

```diff
@@ -4,12 +4,16 @@
 
 from setuptools import setup
 
+# Read the package requirements from the requirements.txt file
+with open('requirements.txt') as f:
+    install_requires = [ line.strip('\n') for line in f.readlines() ]
+
 setup(name = 'picmistandard',
       version = '0.0.13',
```

Comment:
Aren't we at 0.0.14 already?
```suggestion
      version = '0.0.14',
```

Reply:
Fixed by merging in master